server/cli/accounting.go

```diff
@@ -48,37 +48,39 @@ func runGetAcct(cmd *cobra.Command, args []string) {
 		cmd.Usage()
 		return
 	}
-	server.RunGetAcct(Context, args[0])
+	client := client.NewAdminClient(&Context.Context, Context.Addr, client.Accounting)
+	body, err := client.GetYAML(args[0])
+	if err != nil {
```

Comment:
odd. None of permissions/zones call it, even though those are pretty much verbatim copies of accounting.